aapcs64/aapcs64.rst

```diff
@@ -2269,6 +2273,38 @@ a lazy save, with the subroutine having the following properties:
 * The only memory modified by the subroutine (if any) is stack memory
   below the incoming SP.
 
+``__arm_get_current_vg``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+**(Beta)**
+
+Platforms that support SME must provide a subroutine to query the current
+value of VG, with the subroutine having the following properties:
```

Comment:
change to:

and the subroutine must have the following properties:

Reply:
addressed by https://github.com/ARM-software/abi-aa/pull/278